syft/generic/pointers/pointer_plan.py

```diff
@@ -43,18 +47,53 @@ def __init__(
             description=description,
         )
 
+    # Make PointerPlan compatible with multi pointers
+    @property
+    def location(self):
+        n_locations = len(self._locations)
+        if n_locations != 1:
+            return self._locations
+        else:
+            return self._locations[0]
+
+    @location.setter
+    def location(self, new_location: Union[AbstractWorker, List[AbstractWorker]]):
+        if isinstance(new_location, AbstractWorker):
+            self._locations = [new_location]
+        else:
+            self._locations = new_location
+
+    @property
+    def id_at_location(self):
+        n_ids = len(self._ids_at_location)
+        if n_ids != 1:
+            return self._ids_at_location
+        else:
+            return self._ids_at_location[0]
```

Comment:
For all of these would it cause problems if they all returned a list of length 1 and then we just set up the rest of the code to be able to handle that? Throwing this out there, not actually sold on it yet

Reply:
The thing is that I wan the main usecase (as a single pointer) to be unchanged, while enabling another behaviour as a multi pointer